### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by TcgVanguardTroll**
Adding navbar component to portfolio consists of an about me section, education section, work experience section and contact me section.

### CODE REVIEW

File: portfolio/.#pom.xml [hunk 1]
```diff
@@ -0,0 +1 @@
+grantjustice@cs-6000-devshell-vm-edfac7f7-1e25-4780-a66c-aaba2839e09a.547
```
Comment (kmh287) [inline portfolio/.#pom.xml:1]: out of curiosity, what is this file for?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -5,27 +5,121 @@
...
+            </path></svg>
+          <span class="link-text"> About Me </span>
+        </a>
+      </li>
+
+      <li class="nav-item">
+        <a href="#" class="nav-link">
+        <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="university" class="svg-inline--fa fa-university fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M496 128v16a8 8 0 0 1-8 8h-24v12c0 6.627-5.373 12-12 12H60c-6.627 0-12-5.373-12-12v-12H24a8 8 0 0 1-8-8v-16a8 8 0 0 1 4.941-7.392l232-88a7.996 7.996 0 0 1 6.118 0l232 88A8 8 0 0 1 496 128zm-24 304H40c-13.255 0-24 10.745-24 24v16a8 8 0 0 0 8 8h464a8 8 0 0 0 8-8v-16c0-13.255-10.745-24-24-24zM96 192v192H60c-6.627 0-12 5.373-12 12v20h416v-20c0-6.627-5.373-12-12-12h-36V192h-64v192h-64V192h-64v192h-64V192H96z"></path></svg>
```
Comment (kmh287) [inline portfolio/src/main/webapp/index.html]: I like the style above where each attribute was on its own line. Can you do that here as well?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -5,27 +5,121 @@
...
+            focusable="false" 
+            data-prefix="fas" 
+            data-icon="address-card" 
+            class="svg-inline--fa fa-address-card fa-w-18" 
+            role="img" 
+            xmlns="http://www.w3.org/2000/svg" 
+            viewBox="0 0 576 512">
+            <path fill="currentColor" 
+            d="M528 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h480c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48zm-352 96c35.3 0 64 28.7 64 64s-28.7 64-64 64-64-28.7-64-64 28.7-64 64-64zm112 236.8c0 10.6-10 19.2-22.4 19.2H86.4C74 384 64 375.4 64 364.8v-19.2c0-31.8 30.1-57.6 67.2-57.6h5c12.3 5.1 25.7 8 39.8 8s27.6-2.9 39.8-8h5c37.1 0 67.2 25.8 67.2 57.6v19.2zM512 312c0 4.4-3.6 8-8 8H360c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16zm0-64c0 4.4-3.6 8-8 8H360c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16zm0-64c0 4.4-3.6 8-8 8H360c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16z">
+            </path></svg>
```
Comment (kmh287) [inline portfolio/src/main/webapp/index.html:59]: +2 indent

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -5,27 +5,121 @@
...
+            focusable="false" 
+            data-prefix="fab" 
+            data-icon="google" 
+            class="svg-inline--fa fa-google fa-w-16" 
+            role="img" 
+            xmlns="http://www.w3.org/2000/svg" 
+            viewBox="0 0 488 512"><path 
+            fill="currentColor" 
+            d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z">
+    </path>
```
Comment (kmh287) [inline portfolio/src/main/webapp/index.html]: path needs a +2 indent

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -5,27 +5,121 @@
...
+        focusable="false" 
+        data-prefix="far" 
+        data-icon="address-book" 
+        class="svg-inline--fa fa-address-book fa-w-14" 
+        role="img" 
+        xmlns="http://www.w3.org/2000/svg" 
+        viewBox="0 0 448 512"><path 
+        fill="currentColor" 
+        d="M436 160c6.6 0 12-5.4 12-12v-40c0-6.6-5.4-12-12-12h-20V48c0-26.5-21.5-48-48-48H48C21.5 0 0 21.5 0 48v416c0 26.5 21.5 48 48 48h320c26.5 0 48-21.5 48-48v-48h20c6.6 0 12-5.4 12-12v-40c0-6.6-5.4-12-12-12h-20v-64h20c6.6 0 12-5.4 12-12v-40c0-6.6-5.4-12-12-12h-20v-64h20zm-68 304H48V48h320v416zM208 256c35.3 0 64-28.7 64-64s-28.7-64-64-64-64 28.7-64 64 28.7 64 64 64zm-89.6 128h179.2c12.4 0 22.4-8.6 22.4-19.2v-19.2c0-31.8-30.1-57.6-67.2-57.6-10.8 0-18.7 8-44.8 8-26.9 0-33.4-8-44.8-8-37.1 0-67.2 25.8-67.2 57.6v19.2c0 10.6 10 19.2 22.4 19.2z">
+        </path>
```
Comment (kmh287) [inline portfolio/src/main/webapp/index.html]: +2

**Review by kmh287 — APPROVED** [review]
LGTM. Only minor style nits.

File: portfolio/.#pom.xml (hunk 1 above)
Comment (TcgVanguardTroll) [reply portfolio/.#pom.xml:1]: I'm not sure I accidentally went into vim from cloud shell and ended up altering the pom.xml

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (TcgVanguardTroll) [reply portfolio/src/main/webapp/index.html]: Ack

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (TcgVanguardTroll) [reply portfolio/src/main/webapp/index.html:59]: Ack

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (TcgVanguardTroll) [reply portfolio/src/main/webapp/index.html]: Ack

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (TcgVanguardTroll) [reply portfolio/src/main/webapp/index.html]: Ack

**Comment by kmh287** [comment]
Thanks for making the changes Jordan. Going forward please be sure to wait for Tiancong to take a pass over PRs before merging.

**Review by tcwanggoogle — COMMENTED** [review]
LGTM

**Pull request merged**